tests/components/oralb/test_sensor.py

```diff
@@ -59,6 +63,30 @@ async def test_sensors(
     assert await hass.config_entries.async_unload(entry.entry_id)
     await hass.async_block_till_done()
 
+    # Fastforward time without BLE advertisements
+    monotonic_now = start_monotonic + FALLBACK_MAXIMUM_STALE_ADVERTISEMENT_SECONDS + 1
+
+    with patch(
+        "homeassistant.components.bluetooth.manager.MONOTONIC_TIME",
+        return_value=monotonic_now,
+    ), patch_all_discovered_devices([]):
+        async_fire_time_changed(
+            hass,
+            dt_util.utcnow()
+            + timedelta(seconds=FALLBACK_MAXIMUM_STALE_ADVERTISEMENT_SECONDS + 1),
+        )
+        await hass.async_block_till_done()
+
+    assert await hass.config_entries.async_setup(entry.entry_id)
+    await hass.async_block_till_done()
+
+    # All of these devices are sleepy so we should still be available
+    toothbrush_sensor = hass.states.get(
+        "sensor.smart_series_7000_48be_toothbrush_state"
+    )
+    toothbrush_sensor_attrs = toothbrush_sensor.attributes
+    assert toothbrush_sensor.state == "running"
```

Comment:
Shouldn't we have the restart test here as well? it now only tests the timeout

Reply:
added in 0745120c79f8d9e4bfec7b25ebabc4e547decbe8